lib/matplotlib/dates.py

```diff
@@ -211,36 +211,58 @@ def _get_rc_timezone():
 WEEKDAYS = (MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY, SUNDAY)
 
 
-def _to_ordinalf(dt):
+def _to_ordinalf(dts):
```

Comment:
I think it's probably worth writing a new function called something like `_dtime64_to_ordinalf(dts)` with the new code to handle `np.datetime64`s.